msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md

```diff
@@ -153,7 +153,7 @@ The following steps guide you to sideload and validate bot that receives all cha
 
 # [Chat messages](#tab/chat)
 
-The following steps guide you to sideload and validate bot that receives all chat messages without being @mentioned in a chat:
+The following steps guide you to sideload and validate bot that receives all chat messages in [public developer preview]((../../../resources/dev-preview/developer-preview-intro.md)) without being @mentioned in a chat:
 
 1. Select or create a group chat.
```

Comment:
We should add a previous step to switch to developer preview.

Reply:
Added a step for dev preview.